### ISSUE TITLE
Disable navigating samples during continuous sampling

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/projectile-data-lab#37]
The current behavior when trying to navigate samples during continuous sampling is that the sample selector can still navigate back to a previous sample briefly, but is brought back to the most recent sample every time a new one is automatically created. This results in a "fighting" UX when trying to navigate using the sample panel during continuous sampling. 

To fix this, continuous launching mode should be automatically stopped when the user navigates to a previous sample. If someone has decided that they want to look at a previous sample, we want to make sure that data is not also being created "behind the scenes". Note that this is distinct from the VSM screens, since no two samples are ever visible at the same time.

Let's implement an auto-stop for isContinuousLaunching when the sample selector navigates to a previous sample, and keep the behavior that updates the sample selector panel to the most recent sample when a new one is created.

**Comment by samreid** [comment]
Clarifying the proposed design. Let's say the user launches 5 samples. Then navigates back to sample 3. In this case, is the "launch" button enabled? If so, what happens when it is pressed?

**Comment by samreid** [comment]
@matthew-blackman and I worked on the "navigating to previous sample" part.  So this issue still needs to look at the continuous launching part.

**Comment by samreid** [comment]
This design has changed, and now my understanding is that the sample navigation buttons should be grayed out when projectiles are in the air. Should the sample navigation buttons also be grayed out while continuous mode is generating data? I think so but would be good to double check before starting on that part. Also, is this tracked in another issue?

**Comment by samreid** [comment]
OK I disabled the forward/back buttons when continuous launching is on.  @matthew-blackman can you please review/test and look for corner cases?
